lib/main/atom/tooltipManager.ts

```diff
@@ -15,7 +15,10 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any> el).querySelectorAll(selector)
+  if (!found) {
+    found = (<any> el).rootElement.querySelectorAll(selector)
+  }
```

Comment:
I didn't check either, but if they remove `rootElement`, you'll get a type error trying to call `querySelectorAll` on `undefined`.

Just from the PR I can't tell where it's used (this is some legacy code that I'm not familiar with), but my personal preference would be to phase out jQuery altogether. If you set the type of `element` to be `HTMLElement`, you will not need the type casts. This will require updating all of the callsites so idk how much work that would be, but it's worth looking into.